### ISSUE TITLE
TypeError: Cannot read property 'clone' of undefined

### ISSUE THREAD

**Issue by falsandtru** [power-assert-js/karma-espower-preprocessor#3]
https://travis-ci.org/twada/power-assert-keeper/jobs/320195010#L1098

```
22 12 2017 12:32:19.405:ERROR [karma]: TypeError: Cannot read property 'clone' of undefined
    at /home/travis/build/twada/power-assert-keeper/karma-espower-preprocessor/node_modules/karma-espower-preprocessor/index.js:22:29
    at nextPreprocessor (/home/travis/build/twada/power-assert-keeper/karma-espower-preprocessor/node_modules/karma/lib/preprocessor.js:36:26)
    at /home/travis/build/twada/power-assert-keeper/karma-espower-preprocessor/node_modules/karma/lib/preprocessor.js:131:9
    at module.exports (/home/travis/build/twada/power-assert-keeper/karma-espower-preprocessor/node_modules/isbinaryfile/index.js:28:12)
    at readFileCallback (/home/travis/build/twada/power-assert-keeper/karma-espower-preprocessor/node_modules/karma/lib/preprocessor.js:93:7)
    at /home/travis/build/twada/power-assert-keeper/karma-espower-preprocessor/node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
```

**Comment by twada** [comment]
The line 22 is `var opts = helper._.clone(options)`, so `helper._` must be removed on karma master?

**Comment by twada** [comment]
Ah [`helper._` has been marked as deprecated](https://github.com/karma-runner/karma/commit/5c6b151) and removed from [new Karma 2.0.0 release](https://github.com/karma-runner/karma/blob/master/CHANGELOG.md#200-2017-12-21).

**Comment by twada** [comment]
Closed since [v1.1.1](https://github.com/power-assert-js/karma-espower-preprocessor/releases/tag/v1.1.1) is out.
